src/blocks/time.rs

```diff
@@ -70,13 +80,8 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
             unsafe { tzset() };
         }
 
-        let full_time = get_time(format, timezone, locale);
-        let short_time = format_short
-            .map(|f| get_time(f, timezone, locale))
-            .unwrap_or_else(|| "".into());
-
-        widget.set_format(FormatConfig::default().with_defaults(&full_time, &short_time)?);
         widget.set_values(map!("icon" => Value::icon(api.get_icon("time")?)));
+        widget.set_values(map!("timestamp" => Value::timestamp(timezone)));
```

Comment:
```suggestion
        widget.set_values(map! {
            "icon" => Value::icon(api.get_icon("time")?),
            "timestamp" => Value::timestamp(timezone),
        });
```

This resolves

> icon doesn't work

Reply:
Thanks, I noticed that this morning and groaned it my idiocy.